Sources/KituraContracts/CodableQuery/Coder.swift

```diff
@@ -16,21 +16,44 @@
 
 import Foundation
 
-/// Class defining shared resources for the QueryDecoder and QueryEncoder
+/**
+ Class defining shared resources for the QueryDecoder and QueryEncoder.
+ 
+ ### Usage Example: ###
+ ````swift
+ let date = Coder().dateFormatter.date(from: "2017-10-31T16:15:56+0000")!
+ ````
+ */
 public class Coder {
 
-    /// The designated DateFormatter used for encoding and decoding query parameters
+    /**
+     The designated DateFormatter used for encoding and decoding query parameters.
+     
+     ### Usage Example: ###
+     ````swift
+     let date = Coder().dateFormatter.date(from: "2017-10-31T16:15:56+0000")!
+     ````
+     */
     public let dateFormatter: DateFormatter
 
-    /// Initializes a Coder instance with a Date Formatter
-    /// using the "UTC" timezone and "yyyy-MM-dd'T'HH:mm:ssZ" date format
+    /**
+    Initializes a Coder instance with a Date Formatter
```

Comment:
DateFormatter should be all one word. Coder and DateFormatter should also both have backticks around them.